shared/version/api.go

```diff
@@ -160,6 +160,7 @@ var APIExtensions = []string{
 	"resources_v2",
 	"container_exec_user_group_cwd",
 	"container_syscall_intercept",
+	"container_syscall_intercept_mount",
```

Comment:
We only append to this list, so this needs to go at the end and be part of the `api: ...` commit mentioned earlier.